### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by tjwelde** [dim13/otpauth#10] [Show and tell]
Thank you very much for this project!

Not being able to switch to another OTP app, was a pain for a long time for me, because the system I need the OTPs for also has no easy way to setup a new OTP app and Google Authenticator didn't have the functionality to switch to another App.

I found the export functionality in Google Authenticator and looked at the string with a QR reader and found the protocol `otpauth-migration` in it, which led to a google search, which led to this repo, which ultimately led to success!!

Thanks again and have a nice day!

**Comment by dim13** [comment]
I'm glad, it was useful for you.

**Comment by jpluscplusm** [comment]
+1 - thanks for doing this fiddly work!
